SymbolCollisionTest/Podfile

```diff
@@ -6,7 +6,7 @@ target 'SymbolCollisionTest' do
   # use_frameworks!
 
   # Firebase Pods
-    pod 'Firebase', '~> 6.0.0'
+    pod 'Firebase', '6.1.0'
```

Comment:
Should this stick to use `~>`? @paulb777

Reply:
The change is right to use the current version of Firebase.